src/openrct2/rct2/T6Exporter.cpp

```diff
@@ -99,7 +99,10 @@ bool T6Exporter::SaveTrack(OpenRCT2::IStream* stream)
     {
         for (const auto& trackElement : _trackDesign->track_elements)
         {
-            tempStream.WriteValue<uint8_t>(trackElement.type);
+            int8_t trackType = trackElement.type;
```

Comment:
`uint8_t` this is doing a conversion be explicit. `static_cast<uint8_t>(trackElement.type)`